domestic/templates/domestic/guidance_page.html

```diff
@@ -3,7 +3,7 @@
 {% load static from staticfiles %}
 {% load breadcrumbs from component_tags %}
 
-{% block head_title %}{{ page.title }}{% endblock %}
+{% block head_title %}{% firstof page.seo_title page.title%} - great.gov.uk{% endblock %}
```

Comment:
I don't _think_ it'll blow up without a space before the `%` but can't be 100% sure off the top of my head. Either way, worth adding one in for consistency with other tags

```suggestion
{% block head_title %}{% firstof page.seo_title page.title %} - great.gov.uk{% endblock %}
```

Reply:
Ok done